PacketNet.sln

```diff
@@ -15,6 +15,13 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CapturingAndParsingWiFiPack
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ConstructingWiFiPackets", "Examples\ConstructingWiFiPackets\ConstructingWiFiPackets.csproj", "{17699F21-32D6-4511-AFD7-595BB4DCB5C6}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{98EB8373-B1FF-493B-B7E5-3C93780C0070}"

```

Comment:
Is the idea here just to make these files show up in the project? That seems like a good idea, but as a separate commit.

Reply:
Yes, I like to have all files related to a solution/project to be accessible form VS to give a programmer hint that they are present. I am not looking at files directly through file explorer, so I missed those files entirely when I went through the code for the first time.